dtfs-central-api/src/types/utilisation-reports.ts

```diff
@@ -56,12 +56,19 @@ export type UtilisationReportRawCsvData = {
   [HeaderKey in UtilisationReportHeader]: HeaderKey extends `${string}currency` ? Currency : string;
 };
 
+export type FeeRecordReconciledByUser = {
+  firstName: string;
+  lastName: string;
+};
+
 export type FeeRecordPaymentGroup = {
   feeRecords: FeeRecord[];
   totalReportedPayments: CurrencyAndAmount;
   paymentsReceived: Payment[] | null;
   totalPaymentsReceived: CurrencyAndAmount | null;
   status: FeeRecordStatus;
+  reconciledByUser: FeeRecordReconciledByUser | null;
+  dateReconciled: Date | null;
```

Comment:
Where can I see an example of the database structure for a "fee record payment group" and reconciliations please? I'm wondering if these are or should be split up

Reply:
In the database, how are fee records and payments structured please?

Reply:
All entities are listed in `libs/common/src/sql-db-entities`